python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+"""
+module for `vector search` commands.
+"""
+
+from typing import List, Optional, cast
+
+from glide.async_commands.server_modules.ft_constants import (
+    CommandNames,
+    FtCreateKeywords,
+)
+from glide.async_commands.server_modules.ft_options.ft_create_options import (
+    Field,
+    FtCreateOptions,
+)
+from glide.constants import TOK, TEncodable
+from glide.glide_client import TGlideClient
+
+
+async def create(
+    client: TGlideClient,
+    indexName: TEncodable,
+    schema: List[Field],
+    options: Optional[FtCreateOptions] = None,
+) -> TOK:
+    """
+    Creates an index and initiates a backfill of that index.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        indexName (TEncodable): The index name for the index to be created
+        schema (List[Field]): The fields of the index schema, specifying the fields and their types.
+        options (Optional[FtCreateOptions]): Optional arguments for the [FT.CREATE] command.
```

Comment:
why do you add [] every time the commands is mentioned?  can be skipped.
also add : See `FtCreateOptions`.

Reply:
Updated